app/pb_kits/playbook/pb_date_picker/_date_picker.scss

```diff
@@ -0,0 +1,1018 @@
+      }
+    }
+    // Icon for select tags
+    .month-dropdown-icon {
+      align-self: center;
+      margin-top: 5px;
+      width: 10px;
+      position: relative;
+      right: 10px;
+      pointer-events: none;
+    }
+    .year-dropdown-icon {
+      align-self: center;
+      margin-top: 5px;
+      width: 10px;
+      position: relative;
+      right: 12px;
+      pointer-events: none;
+    }
+    // calendar icon styles
+    .cal_icon_wrapper {
+      position: absolute;
+      top: 1px;
+      right: 0px;
+      border-left: 1px solid $border_light;
+      width: 45px;
+      height: 43.2px;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      padding-left: 11px;
+      color: $text_lt_light;
+      &:hover {
+        cursor: pointer;
+      }
+    }
+    .cal_icon {
+      width: 20px;
+    }
+
+    // Header Styles
+    $header_height: 40px;
+
+    .flatpickr-months {
+      // padding: 5px;
+      // padding-bottom: 5px;
+      border-bottom: solid 1px $border_light;
+    }
+    .flatpickr-month {
+      height: $header_height;
+    }
+    .flatpickr-current-month {
+      left: 10px;
+      display: flex;
+      // justify-content: flex-start;
+      padding-top: 0px;
+    }
+    .flatpickr-monthDropdown-months {
+      appearance: none;
+      padding-left: 0px;
+      padding-right: 14px;
+      margin-left: 15px;
+      height: $header_height;
+      &:hover {
+        background-color: white;
+      }
+    }
+    .numInputWrapper {
+      display: contents;
+      .numInput {
+        appearance: none;
+        border-left: solid 1px $border_light;
+        padding-left: 22.5px;
+        padding-right: 14px;
+        margin-left: 10px;
+        height: $header_height;
+        &:focus {
+          outline: none;
+        }
+        &:hover {
+          cursor: pointer;
+        }
+      }
+    }
+
+    .flatpickr-prev-month {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      left: 234px !important;
+      padding-right: 10px;
+      border-left: solid 1px $border_light;
+      height: $header_height;
+      // svg path {
+      //   fill: $neutral;
+      // }
+      &:hover{
+        svg {
+          fill: inherit;
+        }
+      }
+    }
+    .flatpickr-next-month {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      padding-left: 10px;
+      height: $header_height;
+      // svg path {
+      //   fill: $neutral;
+      // }
+      &:hover{
+        svg {
+          fill: inherit;
+        }
+      }
+    }
+
+    div.flatpickr-weekdays {
+      margin-top: 8px;
+    }
+    span.flatpickr-weekday {
+      @include caption;
+      user-select: none;
+    }
+    // Month-Year Select Styles
+    select.flatpickr-monthDropdown-months {
+      font-size: 16px;
+      font-weight: 600;
+      color: #242B42;
+      margin-top: 0px;
+      margin-bottom: 0px;
```

Comment:
Why are these needed? We should be using our text styles. Also variables

Reply:
Switch to `@include pb_title_4`